File: 2_course/ЯПВУ/Laptev_AV1/Лаптев Александр/LaptevAV-1_11.go
package main
import ("fmt"
        "math"
)

func main() {
    var x1 float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&x1)
    var y1 float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&y1)
    var x2 float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&x2)
    var y2 float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&y2)
    st := 0.5
    dl := math.Pow((x2 - x1), 2) + math.Pow((y2 - y1), 2)
    ras := math.Pow(dl, float64(st))
    fmt.Println (ras)
}

File: 2_course/ЯПВУ/Laptev_AV1/Лаптев Александр/LaptevAV-1_11.rb
# encoding utf-8
puts ('Vvedite veshestvennoe chislo: ')
x1 = gets.to_i
puts ('Vvedite veshestvennoe chislo: ')
y1 = gets.to_i
puts ('Vvedite veshestvennoe chislo: ')
x2 = gets.to_i
puts ('Vvedite veshestvennoe chislo: ')
y2 = gets.to_i
ras = ((x2 - x1) ** 2 + (y2 - y1) ** 2) ** 0.5
print ("#{ras}")